Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM golang:1.9-alpine
 
 COPY . /go/src/github.com/honeycombio/honeycomb-opentracing-proxy
 WORKDIR /go/src/github.com/honeycombio/honeycomb-opentracing-proxy
+RUN go test ./...
```

Comment:
I don't think we need to run the tests to build the docker file, was this added for a specific reason?

Reply:
Ah, I added it because I hadn't run the test suite before pushing to the repository, and then the tests failed. I only added it to improve my confidence, and then accidentally included it in the commit that fixed the test failures.

Reply:
I've removed the Dockerfile change now